src/ekore/anomalous_dimensions/unpolarized/time_like/as3.py

```diff
@@ -0,0 +1,791 @@
+"""The unpolarized time-like NNLO Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+from numpy import power as npp
+
+from ....harmonics import w1, w2, w3, w4
+from ....harmonics.constants import zeta2, zeta3
+
+
+@nb.njit(cache=True)
+def gamma_nsp(N, nf):
+    r"""Compute the NNLO non-singlet positive anomalous dimension.
+
+    Based on :cite:`Mitov:2006ic`.
```

Comment:
can you give an equation?

Reply:
```suggestion
    Implements :eqref:`15` from  :cite:`Mitov:2006ic` via the N-space translation from A. Vogt.
```
and similar ... This _is_ the reference if A. Vogt is saying so ... so, we should give him due credit (maybe even add a own bibtex entry?) and also the paper. The fact that it is a different representation is not relevant for the reference here (though it is for the content of the function) ...

Reply:
the reference equation for nsm is Eq. 16: "for both ξ = − and ξ = v. " - to be exact, it is implicit defined there (due to the difference on the lhs). As for the website, I think it is also fine to just say "coming from A. Vogt" people in the field will know ... the only alternative, as said, can be a `misc` bibtex entry ... decide yourself

Reply:
No no, I am not talking about NNLO anymore, for which I have already included the equations as you suggested. I am talking about NLO, where just for the NSP and NSM, I had to use the parametrized expressions from Vogt, however the paper that he mentions on his page only contains NSP and not NSM.

Reply:
then for ns- I would cite "Mitov:2006wy Eq. B.8 with a parametrization provided by A. Vogt" (by the way right above these equations there is again the statement about odd and even moments ...)

Reply:
well, what happened is I had initially taken all expressions from Mitov:2006wy paper, used mathematica to get them in the form which can then be used in python. It appears that the non singlet expressions in the paper might contain some errors and therefor the nsp and nsm were replaced with Vogt's parametrization whereas the qqs, qg, gq, and gg were left as they were (from Mitov's paper). So for nsp and nsm, the citation has to be as per Vogt here, for which as I mentioned above, the paper he mentions does not contain nsm as far as I can tell.

Reply:
fine, I accept as is
PS: last thing: did you see Gluck:1992zx has an erratum?

Reply:
Yes, the erratum does not concern us, as it is for P_gg for which, I use Mitov's expression.